packages/react/src/components/CTA/CTA.js

```diff
@@ -32,188 +24,34 @@ import { VideoPlayerAPI } from '@carbon/ibmdotcom-services';
  */
 const CTA = ({ style, type, customClassName, ...otherProps }) => {
   const [renderLightBox, openLightBox] = useState(false);
-  const [videoTitle, setVideoTitle] = useState([{ title: '', key: 0 }]);
-  const [mediaData, setMediaData] = useState({});
 
-  useEffect(() => {
-    getVideoData();
-  }, [getVideoData, style, type]);
+  const videoTitle = useVideoData(otherProps, style, type);
```

Comment:
We should just pass the videoId in here, and not have the hook reference CTALogic